msdk-io/msdk-io-mzml2/src/test/java/io/github/msdk/io/mzml2/MzMLFileParserTest.java

```diff
@@ -26,20 +25,20 @@
 
 public class MzMLFileParserTest {
 
-	private static final String TEST_DATA_PATH = "src/test/resources/";
+  private static final String TEST_DATA_PATH = "src/test/resources/";
 
-	@Test
-	public void testValidmzMLfile() throws IOException, MSDKException, XMLStreamException {
+  @Test
+  public void testValidmzMLfile() throws IOException, MSDKException, XMLStreamException {
 
-		final String inputFileName = TEST_DATA_PATH + "mzML_with_UV.mzML";
-		final File inputFile = new File(inputFileName);
+    final String inputFileName = TEST_DATA_PATH + "mzML_with_UV.mzML";
+    final File inputFile = new File(inputFileName);
 
-		Assert.assertNotNull(inputFile);
-		MzMLFileParser mzParser = new MzMLFileParser(inputFile);
-		mzParser.execute();
+    Assert.assertNotNull(inputFile);
```

Comment:
What is the reason you check if inputFile == null? Is that ever going to happen?